src/tss2-tcti/tcti-device.c

```diff
@@ -251,6 +294,7 @@ Tss2_Tcti_Device_Init (
     TSS2_TCTI_DEVICE_CONTEXT *tcti_dev;
     TSS2_TCTI_COMMON_CONTEXT *tcti_common;
     const char *dev_path = conf != NULL ? conf : TCTI_DEVICE_DEFAULT;
+    mode_t mode = O_RDWR;
```

Comment:
This isn't a mode, it's a flag since it's used in the second parameter: `int open(const char *pathname, int flags, mode_t mode);`. The reason you're not seeing a type error is that the `flags` paramter is an `int` and `mode_t` maps back to `int`. So this variable should be an `int` and it should be called `flags`.

Reply:
Fixed